frontend/.eslintrc

```diff
@@ -57,12 +54,20 @@
             ]
         }],
         "simple-import-sort/exports": ["error"],
-        "import/no-named-as-default": ["off"] // vite stuff
+        "@typescript-eslint/no-explicit-any": ["error"],
+        "@typescript-eslint/no-floating-promises": ["off"],
+        "react/require-default-props": ["off"], // off since we have typescript instead
+        "@typescript-eslint/no-misused-promises": ["error", { "checksVoidReturn": false }], 
+        "@typescript-eslint/ban-ts-comment": ["off"], // should be removed? - it's here to skip type checking on some files,
+        "@typescript-eslint/dot-notation": ["off"], // disable force use of . annotation when accessing object fields
+        "react-hooks/exhaustive-deps": ["warn"], // Checks effect dependencies
```

Comment:
This should be an error. There is no reason to ever ignore it. if you're getting spammed, you should use callbacks instead.

Reply:
Agree it should be an error. But this PR is getting to big for any refactoring changes LOL. This rule was not enabled before moving to typescript so just added it as a warning for now so we can fix it in a future ticket/PR.